### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by ospillinger**

### CODE REVIEW

File: docs/start.md [hunk 1]
```diff
@@ -1,20 +1,27 @@
 # Get started
 
-## Install the CLI
+## Create a cluster on your AWS account
 
-```bash
+```shell
```
Comment (deliahu) [inline docs/start.md:5]: convert all to `bash`

File: docs/clusters/management/uninstall.md [hunk 2]
```diff
@@ -26,11 +26,6 @@ aws s3 rb --force s3://<bucket>
 aws logs describe-log-groups --log-group-name-prefix=<cluster_name> --query logGroups[*].[logGroupName] --output text | xargs -I {} aws logs delete-log-group --log-group-name {}
 ```
 
-## Delete Certificates
-
-If you've configured a custom domain for your APIs, you can remove the SSL Certificate and Hosted Zone for the domain by
-following these [instructions](networking/custom-domain.md#cleanup).
-
```
Comment (deliahu) [inline docs/clusters/management/uninstall.md:33]: Revisit

File: docs/clusters/observability/metrics.md [hunk 3]
```diff
@@ -80,10 +77,10 @@ you to create a custom metrics from your deployed API that can be later be used
 
 Code examples on how to use custom metrics for each API kind can be found here:
 
-- [RealtimeAPI](../realtime/metrics.md#custom-user-metrics)
-- [RealtimeAPI](../async/metrics.md#custom-user-metrics)
-- [BatchAPI](../batch/metrics.md#custom-user-metrics)
-- [TaskAPI](../task/metrics.md#custom-user-metrics)
+- [RealtimeAPI](../../workloads/realtime/metrics.md#custom-user-metrics)
+- [RealtimeAPI](../../workloads/async/metrics.md#custom-user-metrics)
```
Comment (miguelvr) [inline docs/clusters/observability/metrics.md:81]: RealtimeAPI → AsyncAPI

File: docs/summary.md [hunk 4]
```diff
@@ -2,12 +2,29 @@
...
 ## Clients
 
 * [Install](clients/install.md)
 * [CLI commands](clients/cli.md)
-* [Python API](clients/python.md)
-* [Environments](clients/environments.md)
+* [Python client](clients/python.md)
 * [Uninstall](clients/uninstall.md)
 
 ## Workloads
```
Comment (miguelvr) [inline docs/summary.md:30]: I would put the workloads section before the clients section

**Review by deliahu — APPROVED** [review]

**Pull request merged**